### PR TITLE
Frontend 1701 - Assets in dev-center

### PR DESCRIPTION
**Pull request by bnlb**
Resolves the dev-center side of [FRONTEND-1701](https://algorithmia.atlassian.net/browse/FRONTEND-1701)

### CODE REVIEW

**Comment by langlk** [comment]
Did we intentionally switch some of the dev center typeface to Nunito Sans?

File: _layouts/default.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
     {% endif %}
     {% include open-graph.html %}
     <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/pygments-github.css">
+    <link rel="stylesheet" href="https://cdn.algorithmia.design/1.0.0-rc16/synapse.css">
```
Comment (langlk) [inline _layouts/default.html]: I think we just want our old stylesheets for now, not synapse

Comment (langlk) [reply _layouts/default.html]: Also fairly sure we can remove the synapse font files (Nunito Sans and Roboto Mono)

**Comment by peckjon** [comment]
When do we plan on moving to Synapse?

Have not examined w/ Synapse stylesheet... I know that's the long-term goal
(and no ties to current style other than things don't look terribly weird)

**Comment by langlk** [comment]
> When do we plan on moving to Synapse? Have not examined w/ Synapse stylesheet... I know that's the long-term goal (and no ties to current style other than things don't look terribly weird)

I'm not sure when that's planned, @andryanmiller do you know?

**Comment by peckjon** [comment]
(BTW, attempted to spot-check on that branch, but not seeing much in the way of differences vs the live site... are synapse styles all being overridden, or is my local copy not true?)

**Comment by langlk** [comment]
> (BTW, attempted to spot-check on that branch, but not seeing much in the way of differences vs the live site... are synapse styles all being overridden, or is my local copy not true?)

Hmm, I'm seeing Nunito (our synapse font) on this branch and on test.algorithmia.com/developers, and Rubik (the old font) on algorithmia.com/developers

**Comment by peckjon** [comment]
Thanks. DC on test.algorithmia.com actually looks quite good to me, with the exception of the homepage https://test.algorithmia.com/developers on which the font-size maybe needs to come down to 1rem from 1.25rem... not sure why that one is different from the other pages.

@StephanieKim opinions?

**Comment by langlk** [comment]
> Thanks. DC on test.algorithmia.com actually looks quite good to me, with the exception of the homepage https://test.algorithmia.com/developers on which the font-size maybe needs to come down to 1rem from 1.25rem... not sure why that one is different from the other pages.

I'd recommend not adding the synapse styles alongside the old ones, since we're ending up with both rubik and nunito on test.algorithmia/developers right now

**Comment by peckjon** [comment]
Gotcha -- yes then best to avoid mixing fonts and increasing the weight.

Glad to know the eventual transition will be fairly smooth tho!

File: _layouts/default.html (hunk 1 above)
Comment (bnlb) [reply _layouts/default.html]: Oh perfect, I assumed we'd need them since we're rendering within the context of the web client. Removed!

**Pull request merged**